openebs/buildscripts/docker/Dockerfile.arm64

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#Make the base image configurable. If BASE IMAGES is not provided
+#docker command will fail
+ARG BASE_IMAGE=arm64v8/ubuntu:18.04
```

Comment:
This is the way to define the Dockerfile default value, which will be overwritten by what is passed through the build args. Added this into the Dockerfile, just in case users manually run the docker command without passing the ARG.

Reply:
ok .. got it.